bundler/spec/commands/binstubs_spec.rb

```diff
@@ -288,8 +284,7 @@
         G
 
         bundle "binstubs rack --shebang jruby"
-
-        expect(File.open(bundled_app("bin/rackup")).gets).to eq("#!/usr/bin/env jruby\n")
+        expect(File.readlines(bundled_app("bin/rackup")).first).to start_with("#!/usr/bin/env jruby\n")
```

Comment:
Sorry, last tiny comment. Since we're using `File.readlines` now, can you keep using `eq`?

Reply:
Sorry, I get robotic with reviews.  Done.